app/src/main/java/org/simple/clinic/editpatient/OngoingEditPatientEntry.kt

```diff
@@ -112,47 +117,65 @@ data class OngoingEditPatientEntry @Deprecated("Use the `from` factory function
       numberValidator: PhoneNumberValidator,
       dobValidator: UserInputDateValidator
   ): Set<EditPatientValidationError> {
-    val errors = mutableSetOf<EditPatientValidationError>()
+    return getValidationChecks(alreadySavedNumber, numberValidator, ageOrDateOfBirth, dobValidator)
+        .mapNotNull { check -> check() }
+        .toSet()
+  }
 
-    if (name.isBlank()) {
-      errors.add(FULL_NAME_EMPTY)
-    }
+  private fun getValidationChecks(
+      alreadySavedNumber: PatientPhoneNumber?,
+      numberValidator: PhoneNumberValidator,
+      ageOrDateOfBirth: EitherAgeOrDateOfBirth,
```

Comment:
This parameter is unnecessary since this is already a member of `OngoingEditPatientEntry`.